module/zfs/zfs_znode.c

```diff
@@ -582,6 +582,8 @@ zfs_znode_alloc(zfsvfs_t *zfsvfs, dmu_buf_t *db, int blksz,
 	SA_ADD_BULK_ATTR(bulk, count, SA_ZPL_ATIME(zfsvfs), NULL, &atime, 16);
 	SA_ADD_BULK_ATTR(bulk, count, SA_ZPL_MTIME(zfsvfs), NULL, &mtime, 16);
 	SA_ADD_BULK_ATTR(bulk, count, SA_ZPL_CTIME(zfsvfs), NULL, &ctime, 16);
+	SA_ADD_BULK_ATTR(bulk, count, SA_ZPL_CRTIME(zfsvfs), NULL,
+	    &zp->z_crtime[0], 16);
```

Comment:
I looked into the build failures here. I think this line earlier in the function needs the 11 incremented to 12:
`sa_bulk_attr_t bulk[11];`

Reply:
This would make much more sense than what I thought it was at a glance. Nice catch.